src/unistd.rs

```diff
@@ -2454,6 +2454,7 @@ mod setres {
     use super::{Uid, Gid};
 
     /// Sets the real, effective, and saved uid.
+    /// If one of the arguments equals None, the corresponding value is not changed.
```

Comment:
Best to put a blank line after the first line of the documentation.  Rustdoc will render everything above the first blank as a preview, requiring a click to see the rest.

Reply:
resolved.